OpenLIFUData/Resources/UI/OpenLIFUData.ui

```diff
@@ -106,16 +106,16 @@
            <widget class="QWidget" name="activeSession">
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
-              <widget class="QLabel" name="sessionStatusNameLabel">
+              <widget class="QLabel" name="sessionStatusSubjectNameIdLabel">
                <property name="text">
-                <string>Name:</string>
+                <string>Subject Name (ID)</string>
```

Comment:
Missing colon (:) at the end. Same with session name.

Reply:
And can the width of the label column be increased a bit to accomodate the longer label name.

Reply:
On my screen the entire module widget seems to expand to whatever width it needs to in order for the session status view to fit into it. Maybe that's not the best behavior actually, because a really long subject name causes the entire  module widget to widen. Let me rethink this...